js/ui/control/attribution.js

```diff
@@ -49,6 +49,15 @@ Attribution.prototype = util.inherit(Control, {
             }
         }
 
+        // remove any entries that are duplicates or strict substrings of another entry.
+        attributions.sort();
```

Comment:
> Agreed, but it's also directional, since we're looking for a substring match and want to keep the longer of the two if there's a match.

That's a good point. We should sort by _length_ not alphabetic ordering, then.